lib/user-methods.js

```diff
@@ -24,6 +24,19 @@ function node_require(module) {
     return require(module);
 }
 
+function typeOf(obj) {
+    return ({}).toString.call(obj).match(/\s(\w+)/)[1].toLowerCase();
```

Comment:
I don't know what I was thinking here -- something about types not being available in the arguments array, which doesn't make any sense..